### ISSUE TITLE
Accessibility: aria-label/title attribute missing in ProgressIndicator component internal class where role="progressbar"

### ISSUE THREAD

**Issue by nicgoel** [microsoft/fluentui#17260]
### Environment Information

- **Package version(s)**: 7.88.0
- **Browser and OS versions**: Edge Chromium 88.0.705.81 and Windows 10 Enterprise 20H2

### Describe the issue:
ProgressIndicator control is having progress-bar-name accessibility issue. Image from https://developer.microsoft.com/en-us/fluentui#/controls/web/progressindicator:
![image](https://user-images.githubusercontent.com/63101626/109774020-e571b280-7c25-11eb-9d3e-0be70b58376c.png)

Image from  [Accessibility Insights for Web tool](https://microsoftedge.microsoft.com/addons/detail/accessibility-insights-fo/ghbhpcookfemncgoinjblecnilppimih)
![image](https://user-images.githubusercontent.com/63101626/109773774-97f54580-7c25-11eb-895a-b8f48820a42e.png)

### Please provide a reproduction of the issue in a codepen:
Go to https://developer.microsoft.com/en-us/fluentui#/controls/web/progressindicator.
Run the [Accessibility Insights for Web tool](https://microsoftedge.microsoft.com/addons/detail/accessibility-insights-fo/ghbhpcookfemncgoinjblecnilppimih) on this page.
Issue should be detected by the tool.

#### Actual behavior:

ARIA progressbar doesn't have an accessible name

#### Expected behavior:

No ARIA progressbar name error should appear in the accessibility tool

### Documentation describing expected behavior
[More information about aria-progressbar-name](https://dequeuniversity.com/rules/axe/4.1/aria-progressbar-name?application=msftAI)
[WCAG 1.1.1](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7bd2d2051f-bdc8-4af7-8e18-38aae867e216%7d&cid=6cdcefc2-200a-4a35-b33f-0599ba79a987)

**Comment by gouttierre** [comment]
@nicgoel -Thanks for the feedback! 

@khmakoto - Can you determine if this is a regression or if this behavior is an issue?   cc @smhigley 

To set expectations - Due to the complexity and dependencies of our List components, we are actively trying to minimize risk of regressions. We may not be able to fix this issue with a low risk solution. if a reasonable workaround is available.

**Comment by khmakoto** [comment]
@nicgoel thanks for filing this issue! I can confirm I can repro it and it is actually a bug! I'll have a fix soon!
